### PR TITLE
PUBSTWO-1735/1736 - Springdoc start

### PR DESCRIPTION
**Pull request by dsteinich**

### CODE REVIEW

File: src/main/java/gov/usgs/cida/pubs/webservice/AffliliationMvcService.java [hunk 1]
```diff
@@ -195,7 +193,7 @@ public OutsideAffiliation createOutsideAffiliation(@RequestBody OutsideAffiliati
 		return result;
 	}
 
-	@ApiOperation(value = "", authorizations = { @Authorization(value=PubsConstantsHelper.API_KEY_NAME) })
+//	@ApiOperation(value = "", authorizations = { @Authorization(value=PubsConstantsHelper.API_KEY_NAME) })
```
Comment (ssoper-usgs) [inline src/main/java/gov/usgs/cida/pubs/webservice/AffliliationMvcService.java:196]: These are commented out - can we get rid of them?

File: src/main/java/gov/usgs/cida/pubs/webservice/AuthController.java [hunk 2]
```diff
@@ -25,7 +22,7 @@ public AuthController(@Qualifier("mpPublicationBusService")
 		this.mpPublicationBusService = mpPublicationBusService;
 	}
 
-	@ApiOperation(value = "", authorizations = { @Authorization(value=PubsConstantsHelper.API_KEY_NAME) })
+//	@ApiOperation(value = "", authorizations = { @Authorization(value=PubsConstantsHelper.API_KEY_NAME) })
```
Comment (ssoper-usgs) [inline src/main/java/gov/usgs/cida/pubs/webservice/AuthController.java:25]: and here

File: src/main/java/gov/usgs/cida/pubs/webservice/ContributorMvcService.java [hunk 3]
```diff
@@ -189,7 +187,7 @@ public ContributorMvcService(@Qualifier("corporateContributorBusService") IBusSe
 		return result;
 	}
 
-	@ApiOperation(value = "", authorizations = { @Authorization(value=PubsConstantsHelper.API_KEY_NAME) })
+//	@ApiOperation(value = "", authorizations = { @Authorization(value=PubsConstantsHelper.API_KEY_NAME) })
```
Comment (ssoper-usgs) [inline src/main/java/gov/usgs/cida/pubs/webservice/ContributorMvcService.java:190]: these as well

File: src/main/java/gov/usgs/cida/pubs/webservice/PublicationSeriesMvcService.java [hunk 4]
```diff
@@ -145,7 +143,7 @@ public PublicationSeriesMvcService(@Qualifier("publicationSeriesBusService")
 		return result;
 	}
 
-	@ApiOperation(value = "", authorizations = { @Authorization(value=PubsConstantsHelper.API_KEY_NAME) })
+//	@ApiOperation(value = "", authorizations = { @Authorization(value=PubsConstantsHelper.API_KEY_NAME) })
```
Comment (ssoper-usgs) [inline src/main/java/gov/usgs/cida/pubs/webservice/PublicationSeriesMvcService.java:146]: and here

File: src/main/java/gov/usgs/cida/pubs/webservice/mp/MpListMvcService.java [hunk 5]
```diff
@@ -32,7 +30,7 @@ public MpListMvcService(@Qualifier("mpListBusService") final IBusService<MpList>
 		this.busService = busService;
 	}
 
-	@ApiOperation(value = "", authorizations = { @Authorization(value=PubsConstantsHelper.API_KEY_NAME) })
+//	@ApiOperation(value = "", authorizations = { @Authorization(value=PubsConstantsHelper.API_KEY_NAME) })
```
Comment (ssoper-usgs) [inline src/main/java/gov/usgs/cida/pubs/webservice/mp/MpListMvcService.java:33]: and here, I'll stop commenting but there are others as well that you may or may not want to delete.

**Review by ssoper-usgs — APPROVED** [review]

File: src/main/java/gov/usgs/cida/pubs/webservice/AffliliationMvcService.java (hunk 1 above)
Comment (dsteinich) [reply src/main/java/gov/usgs/cida/pubs/webservice/AffliliationMvcService.java:196]: I left the '@ApiOperation' lines in as comments as they will need to be converted to the new swagger if/when it is activated for secure endpoints.

**Review by mbucknell — APPROVED** [review]

**Pull request merged**